README.md

```diff
@@ -4,12 +4,13 @@ Official Infinyon HTTP Sink connector
 
 ## Sink Connector
 
-HTTP sink connector reads records from data streaming and generates an HTTP  
+HTTP sink connector reads records from data streaming and generates a
```

Comment:
sorry, I updated the README and you'll get a conflict... feel free to overwrite.

Reply:
No worries!